src/main/java/hw6/pages/JdiMetalsColorsPage.java

```diff
@@ -0,0 +1,68 @@
+package hw6.pages;
+
+import com.epam.jdi.light.elements.composite.WebPage;
+import com.epam.jdi.light.elements.pageobjects.annotations.FindBy;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.Css;
+import hw6.DataFromJsonForMetalsColorPage;
+import hw6.entities.Summary;
+import hw6.forms.SummaryForm;
+import org.openqa.selenium.By;
+
+import java.util.List;
+
+public class JdiMetalsColorsPage extends WebPage {
```

Comment:
I don't really understand, but where is the Form for Metals&Colors ?
I expect that you create structure similar to this: Site -> Page -> Form::submit(Metals&Colors)
In the other words, you need to submit Form instead of Page, check LoginForm please.